src/main/java/org/jitsi/jigasi/xmpp/CallControlComponent.java

```diff
@@ -219,6 +219,11 @@ public IQ handleIQ(IQ iq)
         if (callControl == null)
         {
             logger.warn("Call controller not initialized");
+
+            // send service unavailable
+            IQ error = IQ.createResultIQ(iq);
```

Comment:
Yep, when you do iq.setError it sets the type of iq to error and then adds the error (I saw the code from tinder library itself). And what I receive in the client is:

<iq xmlns="jabber:client" type="error" to="" from="successfulbullscommunicateeverywhere@conference.damencho.jitsi.net/focus" id=""><error code="503" type="CANCEL"><service-unavailable xmlns="urn:ietf:params:xml:ns:xmpp-stanzas"/></error></iq>